### PR TITLE
SIM-1894 ADD release announcement for JEP 3.28.0 and JMP 3.4.0

### PR DESCRIPTION
**Pull request by arditymeri**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,6 +6,65 @@ title: jSparrow
 
 ![jSparrow Linebreak Very-Top](/dashboard/img/git-linebreak-very-top.png)
 
+## jSparrow 3.28.0 and jSparrow Maven Plugin 3.4.0 Released
+
+This new jSparrow release brings the final step on transitioning to JUnit 5!
+
+### [Replace JUnit 4 Assertions with JUnit Jupiter](https://jsparrow.github.io/rules/replace-j-unit4-assertions-with-jupiter.html)
```
Comment (sasiePre) [inline docs/README.md:13]: Link not working

File: docs/README.md [hunk 2]
```diff
@@ -6,6 +6,65 @@ title: jSparrow
...
 ![jSparrow Linebreak Very-Top](/dashboard/img/git-linebreak-very-top.png)
 
+## jSparrow 3.28.0 and jSparrow Maven Plugin 3.4.0 Released
+
+This new jSparrow release brings the final step on transitioning to JUnit 5!
+
+### [Replace JUnit 4 Assertions with JUnit Jupiter](https://jsparrow.github.io/rules/replace-j-unit4-assertions-with-jupiter.html)
+
+This rule contributes to a stepwise transition to JUnit 5 by replacing the JUnit 4 assertion methods by the equivalent JUnit 5 ones.
+For more details, visit our [online documentation](https://jsparrow.github.io/rules/replace-j-unit4-assertions-with-jupiter.html).
```
Comment (sasiePre) [inline docs/README.md:16]: Link not working

**Review by sasiePre — CHANGES_REQUESTED** [review]
The two JUnit link does not work, but other than that great work as always!

File: docs/README.md (hunk 1 above)
Comment (arditymeri) [reply docs/README.md:13]: Yes. That will work in the future after we update the documentation space.

**Review by sasiePre — APPROVED** [review]

**Pull request merged**